specs/market-api.yaml

```diff
@@ -1017,6 +1049,10 @@ components:
           type: string
           description: >
             id of the proposal from other side which this proposal responds to
+      required:
+        - state
+        - issuerId
+        - proposalId
```

Comment:
ditto

Reply:
DemandOfferBase can be used as schema for requestBodies/Proposal.